frontend/src/server.ts

```diff
@@ -3,11 +3,15 @@ export interface ChatMessageResponse {
 }
 
 const happyResponse: ChatMessageResponse = {
-  message: 'InferGPT backend is healthy!'
+  message: 'Healthcheck: InferGPT backend is healthy!'
 };
 
 const unhappyResponse: ChatMessageResponse = {
-  message: 'Error: InferGPT backend is unhealthy'
+  message: 'Healthcheck: InferGPT backend is unhealthy'
+};
+
+const unhappyChatResponse: ChatMessageResponse = {
+  message: 'I\'m sorry, but I was unable to get a response from InferGPT. Please check the status of the service using the phrase \"healthcheck\"'
```

Comment:
Bit pinickity but saying "I was unable to get a response from InferGPT" implies separate systems? Isnt the whole thing called InferGPT?